charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -160,20 +158,19 @@ sidecars:
     securityContext:
       readOnlyRootFilesystem: true
       allowPrivilegeEscalation: false
-
+  snapshotController:
+    image:
+      tag: v6.3.2-eks-1-28-9
 proxy:
   http_proxy:
   no_proxy:
-
 imagePullSecrets: []
 nameOverride:
 fullnameOverride:
-
```

Comment:
Don't remove this line

Reply:
`yq` in the update-sidecar-dependencies is linting the file. Is there any particular reason why we shouldn't remove the line?

Reply:
`yq` in the release script is linting the yaml files. Is there any particular reason this line shouldn't be removed?

Reply:
Answered offline*